package.json

```diff
@@ -110,6 +110,7 @@
     "test:watch": "npm run test -- --watch --reporter min",
     "test:perf": "NODE_ENV=production babel-node test/perf",
     "preversion": "npm run clean && npm run build && npm run test:all",
+    "postversion": "echo 'Uninstall react-router to fix npm issue' && npm uninstall react-router",
```

Comment:
This is kind of a hack to do this in "postversion". Ideally, I'd do this in "prepublish", but that would mean that when you clone the repo and run `npm install` locally it will remove the `history` package. During development, or if someone is testing out the examples, I don't want this package the be removed, so that's why I'm doing it on "postversion" where the workflow is:

```
$ npm version patch
$ npm publish
```